server.js

```diff
@@ -20,4 +35,89 @@ app.get("/", function (request, response) {
   response.sendFile(__dirname + "/index.html");
 });
 
-app.listen(process.env.PORT);
+// Get all messages
+app.get("/messages", function (request, response) {
+  response.status(200).json(messages);
```

Comment:
200s are implied in express, so you can just do `response.json(messages)` and it will be a 200 response